calibre.json

```
@@ -0,0 +1,58 @@
+{
+  "Calibre": {
+    "description": "Calibre is an open source e-Book management program, allowing for the management of e-Books and syncing across devices using the content server. This Rock-On is compatible with x86_64/amd64 machines only.</p><p>For more information visit <a href='https://calibre-ebook.com/' target='_blank' rel='noopener noreferrer'>Calibre's Official Site</a>.</p><p><p>Based on the docker image provided by Linuxserver.io: <a href='https://hub.docker.com/r/linuxserver/calibre' target='_blank' rel='noopener noreferrer'>https://hub.docker.com/r/linuxserver/calibre'</a>.</p>",
```

Some changes to align with the architecture and image reference on the other Rockons. Also I recommend to remove the official site reference, since that's covered by the link in the name of the Rockon
```suggestion
    "description": "Calibre is an open source e-Book management program, allowing for the management of e-Books and syncing across devices using the content server. <p>Based on a custom docker image: <a href='https://hub.docker.com/r/linuxserver/calibre' target='_blank'>https://hub.docker.com/r/linuxserver/calibre</a>, available for amd64 and arm64 architecture.</p>",
```